pkg/kubelet/winstats/perfcounter_nodestats.go

```diff
@@ -140,13 +144,32 @@ func (p *perfCounterNodeStatsClient) getMachineInfo() (*cadvisorapi.MachineInfo,
 	}
 
 	return &cadvisorapi.MachineInfo{
-		NumCores:       runtime.NumCPU(),
+		NumCores:       processorCount(),
 		MemoryCapacity: p.nodeInfo.memoryPhysicalCapacityBytes,
 		MachineID:      hostname,
 		SystemUUID:     systemUUID,
 	}, nil
 }
 
+// many of the Windows processor affinity functions will only return the information for a
+// single Processor Group. Since a single group can only hold 64 logical processors, this
+// means when there are more they will be divided into multiple groups.
+// for the above reason, the following function is used to get the cpu number of the windows node.
```

Comment:
```suggestion
// For the above reason, procGetActiveProcessorCount is used to get the cpu count for all processor groups of the windows node.
```

Reply:
Already modified, thanks for your review.

Reply:
Thanks for your contribution!  if you need a reviewer from sig-windows in the future you can find us in the [sig-windows slack](https://kubernetes.slack.com/archives/C0SJ4AFB7)